pawn.json

```diff
@@ -43,7 +44,8 @@
 				"gamemodes/core/player/attachment",
 				"gamemodes/core/admin",
 				"gamemodes/core/systems",
-				"gamemodes/core/commands"
+				"gamemodes/core/commands",
+				"gamemodes/core/Commands"
```

Comment:
duplicate, ``gamemodes/core/Commands`` already exists.

Reply:
actually I have realised the capitcal 'C' of the folder and declared as 'commands` in ``pawn.json``, seems like it's not case-sensitive.

Reply:
idk, in Linux compiler, it is case-sensitive. But if most people uses windows compiler, then we do not need to add it.